library/src/scala/tasty/reflect/CompilerInterface.scala

```diff
@@ -1332,6 +1332,11 @@ trait CompilerInterface {
   /** Fields of a case class type -- only the ones declared in primary constructor */
   def Symbol_caseFields(self: Symbol)(using ctx: Context): List[Symbol]
 
+  /** Default parameters of a case class. Keys are names of the parameters and values –
```

Comment:
Why only for case classes? We should check what happens with methods and other constructors.

Reply:
The implementation works against companions of case classes. I believe for methods, the logic will not work out of the box and will have to be modified.

I would prefer to take it one step at a time: the most trivial implementation first and more complex will follow. With the current implementation, we can already do something porting-wise. We can create an issue for more advanced features like more argument groups (and the discussion of how exactly that should work) or default params of method symbols.

Reply:
Then it should be called `caseClassDefaultParams`.